Packs/PassiveTotal/Scripts/script-PTEnrich.yml

```diff
@@ -78,4 +78,5 @@ dependson:
   - pt-ssl-cert
```

Comment:
I see the script's must commands are not relevant in the v2 integration. Is it still relevant for this pack?

Reply:
@reutshal we simply used command `demisto-sdk update-release-notes -p PassiveTotal -u major`. if you want we can remove the same from ReleaseNotes/2_0_0.md